### PR TITLE
fix: swap JDK to recommend OpenJDK instead

### PR DESCRIPTION
**Pull request by wtrocki**
Replace OracleJDK recommendation with OpenJDK (17). 
Actual description can be reviewed and improved depending on how we want to put recommendation in place.

### CODE REVIEW

**Comment by wtrocki** [comment]
Thinking about it we can be even more neutral and maybe not even mention (17 as number)

File: docs/kafka/kcat-kafka/README.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ endif::[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 * You have a running Kafka instance in {product-kafka}.
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* We recomend using latest LTS version of OpenJDK 17.
```
Comment (wtrocki) [inline docs/kafka/kcat-kafka/README.adoc:77]: I do not think we actually need JDK for kcat. 
```suggestion
```

File: docs/kafka/getting-started-kafka/README.adoc [hunk 2]
```diff
@@ -66,7 +66,7 @@ ifndef::community[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 endif::[]
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* We recomend using latest LTS version of OpenJDK 17.
```
Comment (wtrocki) [inline docs/kafka/getting-started-kafka/README.adoc:69]:
```suggestion
* Latest LTS version of OpenJDK is recommended 
```

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc [hunk 3]
```diff
@@ -75,7 +75,7 @@ ifndef::community[]
 endif::[]
 * You have a running Kafka instance in {product-kafka}.
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* We recomend using latest LTS version of OpenJDK 17.
```
Comment (wtrocki) [inline docs/kafka/kafka-bin-scripts-kafka/README.adoc:78]:
```suggestion
* Latest LTS version of OpenJDK is recommended 
```

File: docs/kafka/quarkus-kafka/README.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ endif::[]
 * You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[VSCode^].
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
 * https://maven.apache.org/[Apache Maven^] 3.6.2 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* We recomend using latest LTS version of OpenJDK 17.
```
Comment (wtrocki) [inline docs/kafka/quarkus-kafka/README.adoc:70]:
```suggestion
* Latest LTS version of OpenJDK is recommended 
```

File: docs/registry/quarkus-registry/README.adoc [hunk 5]
```diff
@@ -72,7 +72,7 @@ endif::[]
 * You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[VSCode^].
 * https://adoptopenjdk.net/[OpenJDK^] 11 or later is installed on Linux or MacOS.
 * https://maven.apache.org/[Apache Maven^] 3.8.x or later is installed (for Quarkus 2.2.x).
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* We recomend using latest LTS version of OpenJDK 17.
```
Comment (wtrocki) [inline docs/registry/quarkus-registry/README.adoc:75]:
```suggestion
* Latest LTS version of OpenJDK is recommended 
```

File: docs/kafka/getting-started-kafka/README.adoc [hunk 6]
```diff
@@ -66,7 +66,7 @@ ifndef::community[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 endif::[]
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* Latest LTS version of OpenJDK is recommended
```
Comment (sterobin) [inline docs/kafka/getting-started-kafka/README.adoc:69]: @wtrocki So sorry for the delay! I somehow missed the reviewer ping. Apologies!

Are you meaning this to be a separate prereq? It seems like an add-on recommendation to the JDK 11 prereq.

So this?

```
* https://adoptopenjdk.net/[JDK^] 11 or later is installed. (Latest LTS version of OpenJDK is recommended.)
```

Or this?

```
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* Latest LTS version of OpenJDK is installed (recommended).
```

Comment (wtrocki) [reply docs/kafka/getting-started-kafka/README.adoc:69]: I do not mind either way. Key is to drop Oracle 🙈  JDK and we have tested latest LTS (17 on windows). 
First option is good IMHO

Comment (sterobin) [reply docs/kafka/getting-started-kafka/README.adoc:69]: @wtrocki Okay, sounds good. If you can apply the first option to all of your cases in the PR, I can merge. Thanks.

**Comment by wtrocki** [comment]
Closing as other PR was created

**Pull request closed without merging**